### PR TITLE
EW-5763 Adding command to generate random secret token

### PR DESCRIPTION
**Pull request by swathimr**
https://track.akamai.com/jira/browse/EW-5763

This reuses the cryptoJS library to generate random secret token

### CODE REVIEW

**Comment by swathimr** [comment]
@sscian @ananner

File: README.md [hunk 1]
```diff
@@ -287,6 +288,18 @@ Usage: `akamai edgeworkers create-auth-token [options] <secretKey>`
 
 4. The `--expiry` value must be between 1 and 60 minutes. The default is `15`.
 
+### Generate a Random Secret Key
+Generates a random secret key that can be used to creating edgeworkers authentication token and in property PMUSER_EW_DEBUG_KEY.
+
+Usage: `node akamai-edgeworkers generate-secret [options]`
```
Comment (dctheobald) [inline README.md:294]: Usage example should not reference node directly.

File: src/cli-main.ts [hunk 2]
```diff
@@ -254,6 +254,32 @@ program
     cliUtils.logAndExit(0, copywrite);
   });
 
+program
+  .command("generate-secret")
+  .description("Generated a random secret key that can be used in the variable PMUSER_EW_DEBUG_KEY in their property and as an input to create auth token using cli command")
```
Comment (dctheobald) [inline src/cli-main.ts:259]: Generates not 'Generated'?

**Review by dctheobald — CHANGES_REQUESTED** [review]
Minor README changes needed; rest of code looks great, thanks!

**Review by dctheobald — APPROVED** [review]
Looks good to me.

**Pull request merged**